pkg/cmd/dev.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/snyk/go-application-framework/pkg/app"
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/cobra"
+)
+
+// Dev is a helper utility for extension authors to define a lightweight CLI to
+// test their extensions outside of the main Snyk CLI.
+func Dev(initializers ...workflow.ExtensionInit) (*cobra.Command, error) {
+	// Initialize the engine with the given workflows
+	logger := log.New(os.Stderr, "", 0)
+	engine := app.CreateAppEngineWithOptions(
+		app.WithLogger(logger),
```

Comment:
Suggestion: In the latest changes on main, we introduced zerolog, so you could make use of zerolog if you like.

Reply:
Thanks! Added in 5e8360f